### PR TITLE
Access queue from frontend

### PR DESCRIPTION
**Pull request by lexysoda**

### CODE REVIEW

**Comment by lexysoda** [comment]
#69

**Review by bratke — APPROVED** [review]
LGTM

**Review by gesundkrank — APPROVED** [review]
LGTM

File: src/main/java/eu/m6r/kicker/api/TournamentApi.java [hunk 1]
```diff
@@ -141,4 +141,28 @@ public void finishTournament(
...
         }
     }
+
+    @POST
+    @Path("queue")
+    @Consumes(MediaType.APPLICATION_JSON)
+    public void addPlayer(final Player player) {
+        try {
+            controller.addPlayer(channelId, player);
+        } catch (PlayerQueue.PlayerAlreadyInQueueException | PlayerQueue.TooManyUsersException
```
Comment (gesundkrank) [inline src/main/java/eu/m6r/kicker/api/TournamentApi.java]: `PlayerQueue.PlayerAlreadyInQueueException` and `PlayerQueue.TooManyUsersException` should result in something else than a 500. Maybe a 400?

**Review by gesundkrank — CHANGES_REQUESTED** [review]
One minor comment

**Review by gesundkrank — APPROVED** [review]

**Pull request merged**